_guide/tools.md

```diff
@@ -26,7 +26,7 @@ A plugin for [retext](https://github.com/retext-project/retext) that scans text
 Another plugin for [retext](https://github.com/retext-project/retext) that checks text content for possible insensitive, inconsiderate language.
 *   [Zettlr: Markdown Editor](https://www.zettlr.com/) - [Zettlr on GitHub](https://github.com/Zettlr/Zettlr) \
 Open source Markdown editor that simplifies the content creation experience by avoiding the clutter seen in many common word processing solutions.
-*   [Drupal](http://drupal.org/) is a very accessible CMS and [these modules can enhance its accessibility](https://accessibility.civicactions.com/guide/resources)
+*   [Drupal](http://drupal.org/) is a very accessible CMS and [these modules can enhance its accessibility](../../guide/resources.md#drupal-accessibility-modules)
```

Comment:
@mgifford close, I think because the resources file is in the same folder the link could be:

```
[these modules can enhance its accessibility](resources.md#drupal-accessibility-modules)
```